docs/source/vscode_ide.md

```diff
@@ -2,19 +2,25 @@
 title: VSCode as Lobster IDE
 ---
 
-Running Lobster programs from within the editor
+VSCode extension
+-------------------
+The vscode extension includes syntax highlithing, 
+a lobster lsp and breakpoints.
+
+See [here](command_line_usage.html) what the lsp is capable of.
+
+### How to install
+-  Download the file `lobster.vsix` found in the github at `docs/vscode/lobster.vsix`.
+
+-  Open your vscode command pallete and search for `Install from VSIX`. Then select the file you just downloaded.
```

Comment:
It's not really "downloading" if most people reading this will already have the Lobster repo